.changeset/bright-boxes-punch.md

```diff
@@ -0,0 +1,7 @@
+---
+"@rainbow-me/rainbowkit": patch
+"example": patch
+"site": patch
+---
+
+Added Valora support with `valorWallet` wallet connector
```

Comment:
@DanielSinclair thanks for merging! this one is missing `a` in `valoraWallet`

Reply:
Fixed